packages/mdx/index.js

```diff
@@ -124,7 +124,7 @@ async function compile(mdx, options = {}) {
 
   const {contents} = await compiler.process(fileOpts)
 
-  return `/* @jsx mdx */
+  return `/* @jsxFrag React.Fragment */
```

Comment:
Maybe something like this?

```suggestion
  return `/** @jsxImportSource mdx */
```